packages/core-utils/src/optimism/deposit-transaction.ts

```diff
@@ -10,6 +10,10 @@ import {
 
 const { hexDataSlice, stripZeros, hexConcat, keccak256, zeroPad } = utils
 
```

Comment:
I pulled the other helpers from ethers internal functions, there arent any for formatting bools so this looks good to me
https://github.com/ethers-io/ethers.js/blob/56fabe987bb8c1e4891fdf1e5d3fe8a4c0471751/packages/transactions/src.ts/index.ts#L118